jax/numpy/lax_numpy.py

```diff
@@ -151,6 +151,8 @@ def _make_scalar_type(onp_scalar_type):
 complex64 = csingle = _make_scalar_type(onp.complex64)
 complex128 = cdouble = _make_scalar_type(onp.complex128)
 
+complex_dtypes = [complex64, complex128]
```

Comment:
These variable names end up exported from `jax.numpy`, so we should avoid polluting the namespace. In general, and underscore prefix on the name will do the trick.

However, in this specific case, I think you might simply write:
`issubdtype(dtype, complexfloating)`
?
You then don't need this definition.

Reply:
Thanks for catching that, changed it to issubdtype.